test-files/ui/features/step_definitions/custom_steps.rb

```diff
@@ -760,3 +760,28 @@
 When(/^I click the review item column in the tag table$/) do
   page.all('td.f1').first.click
 end
+
+Then(/^I should see a node element "([^"]*)" with a selected highlight$/) do |id|
+  el = find('div.layer-data').first('g[class*=" ' + id + '"]')
+  el[:class].include? '.selected'
+end
+
+Then(/^I should see element "([^"]*)" with a yellow highlight$/) do |id|
+  el = find('div.layer-data').first('g[class*=" ' + id + '"]')
+  el[:class].include? '.edited.unsaved'
+end
+
+And(/^I should see an element with class "([^"]*)" on the map$/) do |id|
+  el = find('div.layer-data')
+  el[:class].include? id
```

Comment:
I don't know why this isn't failing, but el[:class] returns `layer-layer layer-data` so I don't think this is a valid assertion